### DISCUSSION TITLE
Blockquotes

### DISCUSSION THREAD

**Discussion by dominikgeimer** [spatie/laravel-markdown#11] [Q&A] [answered]
Hello,

I really like this package, but it seems, that markdown blockquote does not work. 

```
> Lorem ipsum
```

It does not get rendered as `blockquote`. 

Any idea? 

Thanks.

**Comment by freekmurze** [comment]
@riasvdv could you take a look at this one?

**Comment by riasvdv** [comment, marked as answer]
I've added a blockquote to our `it_can_render_markdown` test with the following markdown:

````
# My title

This is a [link to our website](https://spatie.be)

> This is a blockquote

```php
echo 'Hello world';
```
````

and it outputs the following:

```html
<h1 id="my-title">My title</h1>
<p>This is a <a href="https://spatie.be">link to our website</a></p>
<blockquote>
<p>This is a blockquote</p>
</blockquote>
<pre class="shiki" style="background-color: #fff"><code><span class="line"><span style="color: #005CC5">echo</span><span style="color: #24292E"> </span><span style="color: #032F62">&#39;Hello world&#39;</span><span style="color: #24292E">;</span></span>
<span class="line"></span></code></pre>
```

Which results in this:

<h1 id="my-title">My title</h1>
<p>This is a <a href="https://spatie.be">link to our website</a></p>
<blockquote>
<p>This is a blockquote</p>
</blockquote>
<pre class="shiki" style="background-color: #fff"><code><span class="line"><span style="color: #005CC5">echo</span><span style="color: #24292E"> </span><span style="color: #032F62">&#39;Hello world&#39;</span><span style="color: #24292E">;</span></span>
<span class="line"></span></code></pre>

So it seems like that's all working. Is there anything you've changed from the default configurations?

**Reply by dominikgeimer** [reply]
No, I did not change the default configuration.

It just gets rendered as <p>

`<p>> Lorem ipsum</p>`

In my backend I am using Filament Admin and a Markdown field: https://filamentadmin.com/docs/forms/#fields-markdown-editor

Anything works fine, just the blockquote does not get rendered correctly.

**Reply by benjamincrozat** [reply]
If you're doing `<x-markdown>{!! $content !!}</x-markdown>`. `>` is rendered as `&gt;`.
